src/command_modules/azure-cli-devops/setup.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+from codecs import open
+from setuptools import setup
+
+VERSION = '0.0.1'
+
+CLASSIFIERS = [
+    'Development Status :: 3 - Alpha',
+    'Intended Audience :: Developers',
+    'Intended Audience :: System Administrators',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 2',
+    'Programming Language :: Python :: 2.7',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.4',
+    'Programming Language :: Python :: 3.5',
+    'License :: OSI Approved :: MIT License',
+]
+
+DEPENDENCIES = [
+    'azure==2.0.0rc5',
```

Comment:
It doesn't look like you depend on the Azure SDK in this module.
If you do though, depend on the specific components needed (azure-mgmt-*)